### ISSUE TITLE
[Source Request] mangalib.me

### ISSUE THREAD

**Issue by AddanAine** [Paperback-iOS/extensions#110]
Please consider adding this source. Good website for Russian speaking readers.  Thank you.

**Comment by ConradWeiser** [comment]
Hey there!
When I try to navigate to that website, I get the following
![image](https://user-images.githubusercontent.com/9699789/100421467-6bde8d00-3056-11eb-9fb2-f065f2a8d12b.png)

Does this work if you are in Russia? (Is this a geo-lock? Or is the website having other issues at the moment)

**Comment by AddanAine** [comment]
Yeah, work in Russia, that's unfortunate. There is a good alternative readmanga.live . Maybe this one will work.

**Comment by ConradWeiser** [comment]
Unfortunately the VPN service I have doesn't have any servers out in Russia. I don't have a way to take a look at such.

Regarding readmanga.live, I can connect to this one, although after a preliminary look through how the website is formatted, I'm unsure if it's compatible with the source engine we use. I'll take a deeper look later and run it through a checklist to be sure! I haven't had all that much time to be on source development lately, but if it looks compatible, I'll be sure to get it on my list!

**Comment by AddanAine** [comment]
Thank you. 
And Ukrainian vpn will probably work as well, if you have one.
